### PR TITLE
Fix write tests

### PR DESCRIPTION
**Pull request by OlgaEvdokimova**
Fix and write new tests

### CODE REVIEW

File: files/1e7216c1-e7db-477b-804f-a3ddf685c16a/text.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+some file
```
Comment (Fenydude) [inline files/1e7216c1-e7db-477b-804f-a3ddf685c16a/text.txt]: Add to .gitignore this and others

File: pom.xml [hunk 2]
```diff
@@ -54,6 +54,14 @@
             <artifactId>spring-boot-starter-security</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.springframework.security</groupId>
+            <artifactId>spring-security-test</artifactId>
+            <version>5.4.6</version>
```
Comment (Fenydude) [inline pom.xml]: move version to properties

Comment (OlgaEvdokimova) [reply pom.xml]: done

File: files/1e7216c1-e7db-477b-804f-a3ddf685c16a/text.txt (hunk 1 above)
Comment (OlgaEvdokimova) [reply files/1e7216c1-e7db-477b-804f-a3ddf685c16a/text.txt]: done

**Review by ShirshovAlex — DISMISSED** [review]

File: src/test/resources/sql/get-back-initial-data.sql [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+VALUES ('7c0811d5-354b-4ebb-a65c-0b54efc53a80','Viktor','Glush','Viktorovich','viktor@mail.ru','+375448575692','testSkype1','testExp1','testEducation1','testPATH1','testSkills1', 'A2', 'REGISTERED', '5cbe6c4a-0be1-4016-83d9-e9e3222bb908', '21aab798-2d48-459c-bb84-789d2237f933','b38d8113-d5e8-4967-b8ff-e1df0844790f')
+ ON CONFLICT (fm_id) DO NOTHING;
+
+INSERT INTO internship (inship_id, inship_name, inship_start_date, inship_end_date, inship_publication_date, inship_tech_skills, inship_description, inship_requirements, inship_capacity, inship_registration_start_date, inship_registration_end_date, inship_format_name)
+VALUES ('68a051d7-6d82-4879-b0a3-1340e14db54d', 'some name', '2021-07-21T00:00', '2021-09-21T00:00', '2021-07-01T00:00', 'skills1', 'description1', 'requirements1', 300, '2021-07-01T00:00', '2021-07-01T00:00', 'ONLINE') ON CONFLICT (inship_id) DO NOTHING;
+
+INSERT INTO "user_detail" (u_id, u_firstname, u_lastname, u_email, u_login, u_password, u_role)
+VALUES ('b64b3afc-b1be-4c7a-9406-d7d14f436332', 'Maxim', 'Maevsky', 'maevsky@exadel.com', 'maevsky', '1', 'SUPER_ADMIN') ON CONFLICT (u_id) DO NOTHING;
```
Comment (ShirshovAlex) [inline src/test/resources/sql/get-back-initial-data.sql]: Where do you use it and what for?

Comment (OlgaEvdokimova) [reply src/test/resources/sql/get-back-initial-data.sql]: deleted, it was for test. then I changed idea

**Review by ShirshovAlex — DISMISSED** [review]

File: src/test/java/by/exadel/internship/formTest/FormTest.java [hunk 4]
```diff
@@ -2,75 +2,151 @@
...
+        Form form = formRepository.findByIdAndDeletedFalse(id)
+                .orElseThrow(() -> new NotFoundException("Form with uuid = " + id +
+                        " Not Found in DB", "form.uuid.invalid"));
+        assertFalse(form.isDeleted());
+    }
+
+    private UUID uuidOfCreatedAndPostTestDataForm() throws Exception {
+        MockMultipartFile testForm = new MockMultipartFile("form", "form",
+                MediaType.APPLICATION_JSON_VALUE,
+                ("{\"firstName\": \"testName\",\"city\": {\"id\":\"69e6b47a-4c3d-4207-ac2d-801d9eda7ff1\",\"name\":\"Lviv\"}," +
```
Comment (mortal-gpt) [inline src/test/java/by/exadel/internship/formTest/FormTest.java]: Is this json and json from **givenFormWithFile_checkTestData** the same?  If yes, move it to one variable/method

Comment (OlgaEvdokimova) [reply src/test/java/by/exadel/internship/formTest/FormTest.java]: Done

**Review by viktarkolbik — DISMISSED** [review]

**Review by ShirshovAlex — DISMISSED** [review]

**Review by viktarkolbik — APPROVED** [review]

**Review by ShirshovAlex — APPROVED** [review]

**Pull request merged**